ns1/resource_datafeed.go

```diff
@@ -61,6 +64,13 @@ func DataFeedRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ns1.Client)
 	f, _, err := client.DataFeeds.Get(d.Get("source_id").(string), d.Id())
 	if err != nil {
+		// No custom error type is currently defined in the SDK for a data feed.
```

Comment:
feels like we should ticket out making some more error types in the SDK and smooth out this patch?

Reply:
+1

Reply:
Yeah, makes sense, I'll add a ticket for it in the backlog